digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -306,12 +306,19 @@ object ProjectDAO {
   }
 
 
-  def removeProjectLinksById(projectLinkIds: Set[Long])= {
+  def removeProjectLinksById(projectLinkIds: Set[Long], lrmPositionIds: Set[Long]) = {
```

Comment:
We could delete the LRM position ids using a single set parameter: check ids for project links by querying, delete lrm positions by those ids, delete project links by those ids.